### PR TITLE
Fix coding style in Cython files

### PR DESCRIPTION
**Pull request by okuta**
`ignore = W503,W504` is not necessary to format Ctyhon files.

### CODE REVIEW

File: cupy/core/_reduction.pyx [hunk 1]
```diff
@@ -180,7 +180,7 @@ cdef Py_ssize_t _get_contiguous_size(
     return contiguous_size
 
 
-cpdef (Py_ssize_t, Py_ssize_t, Py_ssize_t) _get_block_specs(  # NOQA
+cpdef(Py_ssize_t, Py_ssize_t, Py_ssize_t) _get_block_specs(  # NOQA
```
Comment (leofang) [inline cupy/core/_reduction.pyx]: Is this a typo? I think we need the space...

Comment (okuta) [reply cupy/core/_reduction.pyx]: Thanks, I fixed.

**Comment by toslunar** [comment]
PEP 8 recommends line breaks *before* binary operators, while `W504` is off in the `flake8` defaults.

**Comment by okuta** [comment]
OK, Under the current rules, there are differences between Python and Cython. 
Please let me know if the rules are decided.

**Pull request closed without merging**